src/styles/molecules/card-promo.scss

```diff
@@ -0,0 +1,53 @@
+@use '@/styles/utils/colors';
+
+// ma-card-promo {
+//   @media screen and (min-width: 768px) {
+//     width: 33.33%;
+//   }
+
+//   @media screen and (min-width: 480px) {
+//     width: 100%;
+//   }
+// }
+
+.ma-card-promo {
+  cursor: pointer;
+  position: relative;
+  overflow: hidden;
+  border: 1px solid #DFE2E7;
```

Comment:
based on design system and color palette, `#DFE2E7 ` bisa diganti pake `colors.$neutral-300`;